sky/serve/infra_providers.py

```diff
@@ -31,6 +31,19 @@
     _CONSECUTIVE_FAILURE_THRESHOLD_TIMEOUT // _ENDPOINT_PROBE_INTERVAL)
 
 
+def _interrupt_process_and_children(pid: int) -> None:
+    parent_process = psutil.Process(pid)
+    for child_process in parent_process.children(recursive=True):
+        try:
+            child_process.send_signal(signal.SIGINT)
+        except psutil.NoSuchProcess:
```

Comment:
Is it possible to trigger other error other than `psutil.NoSuchProcess`?

Reply:
The code is borrowed from `sky/utils/subprocess_utils.py` and only made a small change 🤔 Which error do you think will happen?